apps/api/src/services/exchange-rate-data.service.ts

```diff
@@ -83,7 +84,12 @@ export class ExchangeRateDataService {
       factor = this.currencies[`${aFromCurrency}${aToCurrency}`];
     }
 
-    return factor * aValue;
+    if (isNumber(factor)) {
+      return factor * aValue;
+    }
+
+    // Fallback if currencies are not available
```

Comment:
Maybe add a warning to the log, since otherwise people might be confused?

Reply:
Okay, I'm logging an error message now.